source/github-helpers/index.ts

```diff
@@ -113,7 +113,7 @@ export async function isPermalink(): Promise<boolean> {
 
 // Negative function so it can be used directly in `exclude` array
 export function isNotRefinedGitHubRepo(): boolean {
-	return !location.pathname.startsWith('/refined-github/refined-github/');
+	return !location.pathname.startsWith('/refined-github/refined-github');
```

Comment:
I'm not sure if the trailing slash is intentional, but it _is_ a bug so I fixed it 🤷

Reply:
Since we have `asLongAs` now, this function doesn't have to be "negated" btw.

Reply:
Can you change it to a positive function since we dont have to use exclude we can use the new option we added (That I dont remember the name)